src/main/resources/Office365/Office365Config.xml

```diff
@@ -370,6 +395,9 @@
         <value>sites</value>
       </propertiesToShow>
     </property>
+    <property>
+      <scope/>
```

Comment:
Since the "configureGlobally" was set to 1, you should also fill the scope with the appropriate value. If I'm not mistaken, "WIKI" is the equivalent value.

https://www.xwiki.org/xwiki/bin/view/ReleaseNotes/Data/XWiki/13.7RC1/Entry007/